### ISSUE TITLE
[Bug]: Smart Sniffer not detecting drives in HA Green

### ISSUE THREAD

**Issue by mrmamba** [DAB-LABS/smart-sniffer-app#8]
### App Version

0.2.12 

### Home Assistant Version

2026.9.3

### Installation Type

Home Assistant OS (HAOS)

### Hardware

Other (describe below)

### What happened?

Thanks for the great tool :D

Trying to make it work in a HA Green with an SSD via USB (with a cable compatible with all smart commands) and the tool don't detect any drive.

2026/09/20 23:37:10 using smartctl: /usr/sbin/smartctl (version 7.5)
[23:37:10] INFO: SMART Sniffer App v0.2.12
[23:37:10] INFO: Agent: running on port 9099
[23:37:10] INFO: Integration host: 0449a086-smart-sniffer-agent:9099
[23:37:10] INFO: mDNS: smartha-homeassistant
[23:37:10] INFO: Auth: disabled
[23:37:10] INFO: Mock: disabled
[23:37:10] INFO: Web UI: http://localhost:8099 (via ingress)
[23:37:10] INFO: Drives detected: 1
[23:37:10] INFO: ─────────────────────────────────────────
2026/09/20 23:37:10 SMART Sniffer Agent v0.6.2
2026/09/20 23:37:10 smartctl version: 7.5
2026/09/20 23:37:10 Drives detected: 1
2026/09/20 23:37:10 Listening on: 0.0.0.0:9099
2026/09/20 23:37:10 Auth: disabled
2026/09/20 23:37:10 mDNS: enabled
2026/09/20 23:37:10 first poll: using --scan-open for protocol detection, waking drives for SMART baseline
[webui] Serving on port 8099
[webui] Proxying /api/* → 127.0.0.1:9099
[webui] Proxying /mock/* → 127.0.0.1:9100
2026/09/20 23:37:11 cache refreshed: 1 drive(s)
2026/09/20 23:37:11 mDNS: interfaces: eth0
2026/09/20 23:37:11 mDNS: using custom instance name: smartha-homeassistant
2026/09/20 23:37:11 mDNS: preferred IP: 172.30.33.3
2026/09/20 23:37:11 mDNS: advertising smartha-homeassistant._smartha._tcp.local. on port 9099
[webui] 172.30.32.2 - "GET // HTTP/1.1" 200 -
[webui] 172.30.32.2 - "GET //api/health HTTP/1.1" 200 -
[webui] 172.30.32.2 - code 502, message Backend unavailable on port 9100
[webui] 172.30.32.2 - "GET //mock/health HTTP/1.1" 502 -
[webui] 172.30.32.2 - "GET //api/health HTTP/1.1" 200 -
[webui] 172.30.32.2 - "GET //api/drives HTTP/1.1" 200 -
[webui] 172.30.32.2 - code 502, message Backend unavailable on port 9100
[webui] 172.30.32.2 - "GET //mock/health HTTP/1.1" 502 -
[webui] 172.30.32.2 - "GET //api/drives HTTP/1.1" 200 -
[webui] 172.30.32.2 - "GET //api/drives/s12rneacc19042a HTTP/1.1" 200 -
[webui] 172.30.32.2 - "GET //api/drives/s12rneacc19042a HTTP/1.1" 200 -
[webui] 172.30.32.2 - code 502, message Backend unavailable on port 9100
[webui] 172.30.32.2 - "GET //mock/health HTTP/1.1" 502 -
[webui] 172.30.32.2 - code 502, message Backend unavailable on port 9100
[webui] 172.30.32.2 - "GET //mock/health HTTP/1.1" 502 -
[webui] 172.30.32.2 - "GET //api/health HTTP/1.1" 200 -
[webui] 172.30.32.2 - "GET //api/health HTTP/1.1" 200 -
[webui] 172.30.32.2 - "GET //api/drives HTTP/1.1" 200 -
[webui] 172.30.32.2 - "GET //api/drives HTTP/1.1" 200 -
[webui] 172.30.32.2 - "GET //api/drives/s12rneacc19042a HTTP/1.1" 200 -
[webui] 172.30.32.2 - "GET //api/drives/s12rneacc19042a HTTP/1.1" 200 -

Don't know that IP from where is, is not of my internal network 

### App Logs

```shell

```
